packages/integration-tests/utils/fixtures.ts

```diff
@@ -11,7 +11,13 @@ const getAsset = (assetDir: string, asset: string): string => {
     return assetPath;
   }
 
-  return `${path.dirname(assetDir)}/${asset}`;
+  const upperDirAssetPath = `${path.dirname(assetDir)}/${asset}`;
```

Comment:
What do you man here by "upper dir?"

Reply:
Ah, it's my weird naming for the parent directory that we fall back if the asset is not defined for a case, but defined for a case group. 

Renaming to `parentDirAssetPath`